client/my-sites/checkout/composite-checkout/lib/translate-cart.ts

```diff
@@ -129,6 +129,13 @@ export function translateResponseCartToWPCOMCart( serverCart: ResponseCart ): WP
 		},
 	};
 
+	const alwaysEnabledPaymentMethods = [ 'full-credits', 'free-purchase' ];
+
+	const allowedPaymentMethods = [ ...allowed_payment_methods, ...alwaysEnabledPaymentMethods ]
```

Comment:
As discussed in Slack, I'm ok with the duplication because this list is only used to filter payment methods (ie: `allowedPaymentMethods.includes(...)`.